### PR TITLE
Upgrade to Ghidra 10.2

### PR DESCRIPTION
**Pull request by TSRBerry**
This PR allows this plugin to be compiled for 10.2 and import files without issues.

Although I tested it, I can't really guarantee that this works as expected. To me these changes look pretty minor and shouldn't cause issues, but I'd rather have someone take a look and verify everything works as expected.

### CODE REVIEW

**Comment by gibbed** [comment]
Thanks. I'm lazy so I built this PR using GitHub Codespaces. [SwitchLoader-1.4.0-c79b822-Ghidra_10.2.zip](https://github.com/Adubbz/Ghidra-Switch-Loader/files/9984676/SwitchLoader-1.4.0-c79b822-Ghidra_10.2.zip)

File: src/main/java/adubbz/nx/loader/common/MemoryBlockHelper.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ private void addUniqueSection(String name, long addressOffset, long offset, long
             newBlockName = name + "." + nameCounter; 
         }
         
-        Msg.info(this, "Adding unique section " + newBlockName + " from " + startAddr.toString() + " to " + endAddr.toString());
+        Msg.info(this, "Adding unique section " + newBlockName + " from " + startAddr + " to " + endAddr.toString());
```
Comment (GGLinnk) [inline src/main/java/adubbz/nx/loader/common/MemoryBlockHelper.java:70]: Did you miss that ".toString()`` or did you leave it in on purpose?

Comment (TSRBerry) [reply src/main/java/adubbz/nx/loader/common/MemoryBlockHelper.java:70]: I missed it, thank you! :D

**Comment by GGLinnk** [comment]
I suggest you to update the README since Java 17 is required

**Comment by GGLinnk** [comment]
Latest build :

[SwitchLoader-1.4.0-b7e0163-Ghidra_10.2.2.zip](https://github.com/Adubbz/Ghidra-Switch-Loader/files/10032413/SwitchLoader-1.4.0-b7e0163-Ghidra_10.2.2.zip)

**Review by GGLinnk — APPROVED** [review]
So far, my tests show no errors and no vulnerabilities or intentional flaws have been found.
It seems that everything is fine.

**Pull request merged**